web/src/features/app-sidebar/AppSidebar.tsx

```diff
@@ -44,6 +45,11 @@ const MENU_ITEMS = [
     to: `${PORTAL_URL}/datasets`,
     icon: FileDownIcon,
   },
+  {
+    label: 'Data Explorer',
+    to: `${PORTAL_URL}/data-explorer`,
+    icon: ChartNoAxesCombined,
```

Comment:
This is a different icon to the portal, we should update both at the same time to keep it consistent! (if this is the icon we want)